lib/rouge/lexers/fortran.rb

```diff
@@ -7,11 +7,11 @@ module Rouge
   module Lexers
     class Fortran < RegexLexer
       title "Fortran"
-      desc "Fortran 95 Programming Language"
+      desc "Fortran 2008 Programming Language"
```

Comment:
I think it should be indicated that F08, F03, F95 and F90 are actually supported. At least when looking at the filenames below:
`Fortran 90 to 2008`